windows-installer/stackdriver_logging_agent.nsi

```diff
@@ -0,0 +1,311 @@
+;--------------------------------
```

Comment:
I think there's value in having a consistently named entry point... Let's rename this to setup.nsi.